components/resource_control/src/config.rs

```diff
@@ -0,0 +1,97 @@
+// Copyright 2024 TiKV Project Authors. Licensed under Apache-2.0.
+use std::{fmt, sync::Arc};
+
+use online_config::{ConfigManager, ConfigValue, OnlineConfig};
+use serde::{Deserialize, Serialize};
+use tikv_util::config::VersionTrack;
+
+#[derive(Clone, Serialize, Deserialize, PartialEq, Debug, OnlineConfig)]
+#[serde(default)]
+#[serde(rename_all = "kebab-case")]
+pub struct Config {
+    #[online_config(skip)]
+    pub enabled: bool,
+    pub priority_ctl_strategy: PriorityCtlStrategy,
+}
+
+impl Default for Config {
+    fn default() -> Self {
+        Self {
+            enabled: true,
+            priority_ctl_strategy: PriorityCtlStrategy::Moderate,
+        }
+    }
+}
+
+#[derive(Clone, Copy, Default, Debug, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(rename_all = "kebab-case")]
+pub enum PriorityCtlStrategy {
+    Aggressive,
```

Comment:
Can you provide some comments for different option?

Reply:
Done.